modules/base/manifests/mounts.pp

```diff
@@ -10,7 +10,9 @@
 class base::mounts(
   $assets_disks,
   $graphite_disks,
+  $cdnlogs_disks,
```

Comment:
This is a mandatory parameter so I think it'll have to be set in ` hieradata/node.backup0.backup.provider1.production.govuk.service.gov.uk.yaml
`